### ISSUE TITLE
[FadeInImage] - Does not support Accessibility Text (semanticLabel)

### ISSUE THREAD

**Issue by localpcguy** [flutter/flutter#26040]
The FadeInImage Widget should support `semanticLabel` for use to increase accessibility for images loaded using this widget. This seems like it could be an issue that is good for new contributors maybe? Let me know if a PR would be useful.

**Comment by jonahwilliams** [comment]
Hi @localpcguy, this would be a welcome contribution. It should probably match the implementation of the existing image widget, though you might want to allow the placeholder to have separate semantics

**Comment by goderbauer** [comment]
In the meantime you should be able to just wrap your FadeInImage widget with a Semantics widget.

**Comment by localpcguy** [comment]
Yup, @goderbauer, that's what I did. Thanks.
